b2/download_dest.py

```diff
@@ -59,6 +59,7 @@ class DownloadDestLocalFile(AbstractDownloadDestination):
     """
     Stores a downloaded file into a local file and sets its modification time.
     """
+    MODE = 'wb+'
```

Comment:
How about a comment?  Something like "Truncate and open for writing"

Reply:
I dare not explain the full consequences of using that in a single-line comment. This, as a lot of other things we are using, is documented elsewhere.

Also, a comment here would make this line prone to being half-updated and thus not accurate, someone could easily lose data over this. I had consciously chosen to force the reader into looking it up, because it is too dangerous to attempt a shortcut.